blog/_posts/2017-09-04-clojurians-update.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Clojurian's Update - Working on re-frame's Tracing Tool
+layout: post
+created_at: Mon September 4 2017
+permalink: blog/2017-09-04-clojurians-update.md 
+current: blog
+author: Saskia and Chris
+twitter: saskalii
+categories:
+- student-posts
+---
+
+
+# Really Great Summer of Clojure
+
+It's really exciting to see and feel our progress. In the beginning we were grasping at the huge space of ClojureScript tools and libraries and editors, and now as the last month begins our contributions to re-frame’s tracing tool have already made it easier for developers to debug their applications! There's also a hint of melancholy that the summer of code will be ending in four weeks. Thoughts are starting to take shape about how to address the last third of the project. What do we want to achieve? What would be the ideal outcome? Let's gather some thoughts!
+
+## Learning and Growing
+
+Through a glorious combination of working on re-frame's tracing tool, reading other peoples' open source projects, hanging out with our wonderful mentors and coaches, and tinkering with our own side projects, we gathered experience in programming with ClojureScript and re-frame and learned about langugage specific idioms. What I find most valuable though is the great advice we got from our coaches and our mentor. They all have given us so much input for how to become good programmers, not only technically, but especially in terms of finding a good and efficient workflow, addressing problems on a conceptual level and the importance of clear and constructive communication. 
+
+[coaching with Matt!](/img/blog/2016/coaching.jpg)
+*photo taken by Chris Sun*
```

Comment:
You would need to use image credit class for the images. You can check out the other posts for an example. Can you please change this for all the images?